src/main.c

```diff
@@ -1,7 +1,15 @@
+#include "setup.h"
+
 /**
  * @brief Empty main
  */
 int main(void)
 {
+
+  setup();
+
+  // FIXME: kinematic configuration goes here
+  
+  systick_interrupt_enable();
```

Comment:
This function should go inside setup()

Reply:
Currently, yes, but later on we will need to do some things between setup and enabling the systick.

Maybe makes sense enable it on setup and disabling it inmediatly back in main if this way setup does performs all the functionality intended...

Reply:
Anyway, this function shouldn't be here, as it  is defined in libopencm3, not in HAL.